config/locales/en/activemodel.yml

```diff
@@ -156,9 +156,9 @@ en:
               inclusion: Select yes if your client receives student finance
             student_finance_amount:
               blank: Enter the total amount of student finance this academic year
-              too_many_decimals: &student_error Student loan amount must be an amount of money, like 10,000
-              not_a_number: *student_error
-              greater_than_or_equal_to: *student_error
+              too_many_decimals: Student loan amount cannot include more than 2 decimal places
+              not_a_number: Student loan amount must be an amount of money, like 60,000
+              greater_than_or_equal_to: Student loan amount must 0 or more
```

Comment:
These errors have wording that are inconsistently handled across the app. Some use this wording ^^, while others repeat the same wording (as this did) - `Student loan amount must be an amount of money, like 10,000`. 

I am not sure what we should be using, but would be good to be consistent!

Reply:
Feels like Quay should be involved!

Reply:
have fallen back to the old messages. separate ticket needed to apply uniform message across the app, following designs input